.github/workflows/build-admin-ui.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Build
         uses: actions/setup-node@v3
         with:
-          node-version: '16.x'
+          node-version: 16
         if: steps.node-modules-cache.outputs.cache-hit != 'true'
```

Comment:
I think these also need to be removed in this file or else the [build](https://github.com/hashicorp/boundary-ui/actions/runs/4236321296/jobs/7361104771) will also fail.

Reply:
That is correct, changes applied.